wiki/Client/Interface/Multiplayer/en.md

```diff
@@ -6,7 +6,7 @@ needs_cleanup: true
 
 # Multiplayer
 
-**Multiplayer** (sometimes shortened to *Multi*) is a mode in which up to 16 players can compete against each other or as a team on maps decided by the host.
+**Multiplayer** (sometimes shortened to *Multi*) is a mode in which up to 16 players can compete against each other or as a team on maps decided by the host. Players can also complete maps together depending on host's settings.
```

Comment:
would rather have this combined into one sentence if it absolutely must be mentioned

```suggestion
**Multiplayer** (sometimes shortened to *Multi*) is a mode in which up to 16 players can compete against each other individually or in teams, or play together co-operatively, on maps decided by the host.
```